wrangler-api/src/main/java/io/cdap/wrangler/api/Row.java

```diff
@@ -37,7 +38,10 @@
   // Values held by the row.
   private List<Object> values = new ArrayList<>();
 
+  private final UUID id;
```

Comment:
it doesn't look like the id is used anywhere. It would be better to add it later if required instead of right now.